MoEPluginBase/Manager/MoEPluginInitializer.h

```diff
@@ -0,0 +1,23 @@
+//
+//  MoEPluginInitializer.h
+//  MoEPluginBase
+//
+//  Created by Chengappa C D on 14/02/20.
+//  Copyright © 2020 MoEngage. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UserNotifications/UserNotifications.h>
+#import <UIKit/UIKit.h>
+#import <MoEngage/MoEngage.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MoEPluginInitializer : NSObject
```

Comment:
Let's have a generic Name for this Class, not `MoEPluginInitializer` will act as an interface class for the clients for any of the exposed methods. Similar to what we have for other modules, so that its not just limited to Initialization function. We can discuss the same tomorrow.

Reply:
Lets name it `MoEPlugin`